exp-player/addon/components/isp-radio-group/template.hbs

```diff
@@ -2,21 +2,37 @@
     {{#each options as |option index|}}
       <label class="text-center label-text {{if formatLabel formatLabel}}">
           {{#if labelTop}}
-            {{t option}}
+            {{#if hiddenOptions}}
+              {{#each hiddenOptions as |hidden|}}
+                <span class="{{if (eq hidden option) 'hidden'}}">{{t option}}</span>
+              {{/each}}
+            {{else}}
+                {{t option}}
+            {{/if}}
             {{radio-button value=(last-segment option) checked=value}}
             {{#each-in labels as |ratings value|}}
               {{#if (eq value.rating option)}}
-                {{t value.label}}
+                <span class="{{if value.formatClass value.formatClass}}">
+                    {{t value.label}}
+                </span>
               {{/if}}
             {{/each-in}}
           {{else}}
             {{#each-in labels as |ratings value|}}
               {{#if (eq value.rating option)}}
-                {{t value.label}}
+                <span class="{{if value.formatClass value.formatClass}}">
+                    {{t value.label}}
+                </span>
               {{/if}}
             {{/each-in}}
             {{radio-button value=(last-segment option) checked=value}}
-            {{t option}}
+            {{#if hiddenOptions}}
```

Comment:
That's what I did initially, but if the span isn't there it throws off the alignment of the radio buttons